exercises/pascals-triangle/src/example/java/TriangleGenerator.java

```diff
@@ -0,0 +1,21 @@
+class TriangleGenerator {
```

Comment:
Maybe `PascalsTriangleGenerator` would make it more obvious what the class is for?